packages/@uppy/core/src/index.js

```diff
@@ -30,7 +30,7 @@ class Uppy {
   /**
    * Instantiate Uppy
    *
-   * @param {Object} opts — Uppy options
+   * @param {object} opts — Uppy options
```

Comment:
I thought the new jsdoc rule set actually wanted these capitalized? 🤔

Reply:
could be, it might've changed in one of the 10(!) recent major bumps in the jsdoc plugin :laughing: